adapters/krushmedia/krushmedia.go

```diff
@@ -0,0 +1,208 @@
+package krushmedia
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type KrushmediaAdapter struct {
+	endpoint template.Template
+}
+
+func NewKrushmediaBidder(endpointTemplate string) *KrushmediaAdapter {
+	template, err := template.New("endpointTemplate").Parse(endpointTemplate)
+	if err != nil {
+		return nil
+	}
+	return &KrushmediaAdapter{endpoint: *template}
+}
+
+func (a *KrushmediaAdapter) CheckHasImps(request *openrtb.BidRequest) error {
+	if len(request.Imp) == 0 {
+		err := &errortypes.BadInput{
+			Message: "Missing Imp Object",
+		}
+		return err
+	}
+	return nil
+}
+
+func GetHeaders(request *openrtb.BidRequest) *http.Header {
```

Comment:
This seems like a helper method too. Any reason for it being an exported/public method vs keeping it private?